nilearn/datasets/atlas.py

```diff
@@ -112,7 +123,10 @@ def fetch_atlas_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True
 
     # Download the zip file, first
     files_ = _fetch_files(data_dir, files, verbose=verbose)
-    labels = np.recfromcsv(files_[0])
+    labels = pd.read_csv(files_[0])
+    if legacy_format:
+        warnings.warn(_LEGACY_FORMAT_MSG)
+        labels = labels.to_records(index=False)
```

Comment:
great, thanks! we still need to convert column names to lower-case

Reply:
Thanks @jeromedockes ! I think the other fetchers are OK.
I will take another look to verify I didn't forget something though.

Reply:
Great, thanks! apart from that last detail the PR LGTM :)